core/src/main/python/wlsdeploy/aliases/alias_entries.py

```diff
@@ -898,12 +858,24 @@ def _unit_test_only_get_category_map_files(self):
         :return: blob of stuff
         """
         result = {}
-        for key, value in self.__model_categories_map.iteritems():
-            category_file_name = '%s.json' % value
+        for key in self.__all_model_categories:
+            category_file_name = '%s.json' % self._get_category_file_prefix(key)
             category_file_path = '%s%s' % (self.__category_modules_dir_name, category_file_name)
             result[key] = category_file_path
         return result
 
+    def _get_category_file_prefix(self, category_name):
+        """
+        Return the file prefix for the specified top-level category.
+        The file prefix should match the category name exactly.
```

Comment:
That's interesting about Application. Application and AppDeploymentare the same MBean  in WLST. Not sure if there is a reason for using Application over AppDeployment.

Reply:
I think this one was in place before the rule that the alias file name should match the WLST folder exactly.  Correcting it now would break current models.  We could deprecate and support both for a while, but not sure it's worth it.

Reply:
This could be corrected by changing the alias file name, just didn't want to clutter up the review some more.